e2e/token-vault-app/src/main.ts

```diff
@@ -115,33 +119,40 @@ const res = await (async () => {
 })();
 if (res.hasTokens) {
   loggedInEl.innerText = 'true';
-  hasTokensEl.innerText = 'true';
 }
 
 /** ****************************************************
  * ATTACH USER EVENT LISTENERS
  */
-fetchMockBtn.addEventListener('click', async (event) => {
+fetchProtectedMockBtn.addEventListener('click', async (event) => {
   await fetch('https://jsonplaceholder.typicode.com/todos');
 });
+
+fetchUnprotectedMockBtn.addEventListener('click', async (event) => {
```

Comment:
Wanted to test requests that are not allowlisted to ensure they are not intercepted.